### PR TITLE
DOC-425 Adding drafts for ABSOLUTE LiftOver, Aliquot Ensemble Somatic…

### PR DESCRIPTION
**Pull request by ebusygina**
Adding drafts for ABSOLUTE LiftOver, Aliquot Ensemble Somatic Variant Merging and Masking, Arriba, ASCAT2, ASCAT3 and AscatNGS workflows

### CODE REVIEW

File: docs/Encyclopedia/pages/ABSOLUTE_LiftOver.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ABSOLUTE LiftOver
+
+## Description ##
+
+ABSOLUTE LiftOver is a copy number variation (CNV) pipeline used in GDC genotyping array harmonization.
+
+## Overview ##
+
+The ABSOLUTE LiftOver workflow in the GDC is a copy number variation (CNV) pipeline used for genotyping array harmonization. It converts genomic coordinates from hg19 to GRCh38, ensuring high-quality, curated CNV data for downstream analysis. This pipeline also provides gene-level copy numbers, along with purity and ploidy measurements.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/ABSOLUTE_LiftOver.md:9]: It took me forever to verify this was from hg19, great job tracking this information down!

File: docs/Encyclopedia/pages/ASCAT2.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# ASCAT2
+
+## Description ##
+
+ASCAT2 is a copy number variation (CNV) analysis pipeline used in GDC genotyping array harmonization.
+
+## Overview ##
+
+ASCAT2 is used to analyze CNVs in tumor and normal samples from genotyping arrays. It processes the CEL files to generate segmented CNV data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/ASCAT2.md:9]: s/the CEL files/raw array files/

File: docs/Encyclopedia/pages/ASCAT3.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# ASCAT3
+
+## Description ##
+
+ASCAT3 is an advanced copy number variation (CNV) analysis pipeline used in GDC genotyping array harmonization.
+
+## Overview ##
+
+ASCAT3 improves upon previous versions to provide more accurate CNV detection in tumor and normal samples from genotyping arrays. It processes the CEL files to generate high-resolution segmented CNV data.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/ASCAT3.md:9]: Same as above

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Aliquot Ensemble Somatic Variant Merging and Masking
+
+## Description ##
+
+Aliquot Ensemble Somatic Variant Merging and Masking is a somatic variant calling pipeline used in GDC whole exome sequencing (WXS) and targeted sequencing harmonization.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md:5]: Technically somatic variants aren't called, so I would recommend changing it from "a somatic variant calling pipeline" to "an aggregation pipeline"

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## Overview ##
+
+The Aliquot Ensemble Somatic Variant Merging and Masking workflow in the GDC harmonizes whole exome sequencing (WXS) and targeted sequencing data by identifying and merging somatic variants. This workflow consists of Somatic Aggregation and Masked Somatic Aggregation, which respectively generate comprehensive mutation annotation files (MAFs) and produce filtered, public-access MAFs by removing potentially identifiable germline mutation information.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md:9]: Change "by identifying and merging somatic variants." to "by merging multiple MAFs that were each generated by one variant caller."

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+### Input
+
+* Tumor MAF - WXS or Targeted Sequencing
+* Normal MAF - WXS or Targeted Sequencing
+
+(?) question about input data formats: on the portal I see only MAF input files, in [docs](/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/#somatic-aggregation-workflow) I see VCF for Somatic Aggregation Workflow
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md:16]: I think we discussed this, but feel free to leave this as just MAFs

File: docs/Encyclopedia/pages/Arriba.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+# Arriba
+
+## Description ##
+
+The Arriba workflow is part of the GDC mRNA quantification analysis pipeline used for detecting gene fusions in RNA-Seq data. 
+
+## Overview ##
+
+Arriba is one of the two pipelines the GDC uses to detect gene fusions. The Arriba gene fusion pipeline uses Arriba v1.1.0 to detect gene fusions from the RNA-Seq data of tumor samples.
```
Comment (wwysoc2) [inline docs/Encyclopedia/pages/Arriba.md:9]: We might want to shy away from specific versions here, in case they change.

**Review by wwysoc2 — CHANGES_REQUESTED** [review]

File: docs/Encyclopedia/pages/ASCAT2.md (hunk 2 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/ASCAT2.md]: changed

File: docs/Encyclopedia/pages/ASCAT3.md (hunk 3 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/ASCAT3.md]: changed

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md (hunk 4 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md]: changed

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md (hunk 5 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md]: changed

File: docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md (hunk 6 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Aliquot_Ensemble_Somatic_Variant_Merging_and_Masking.md]: deleted :)

File: docs/Encyclopedia/pages/Arriba.md (hunk 7 above)
Comment (ebusygina) [reply docs/Encyclopedia/pages/Arriba.md]: changed

**Review by wwysoc2 — APPROVED** [review]
lgtm, merge into dev whenever you are ready

**Pull request merged**